src/components/sandboxLink.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import usePlatform from "./hooks/usePlatform";
+
+type Props = {
+    children: React.ReactNode;
+    scenario: string;
+    projectSlug: string;
+    errorType: string;
```

Comment:
@Neo-Zhixing Most of these props should be optional, yea?

Reply:
right